prow/plugins/project/project_test.go

```diff
@@ -227,8 +227,8 @@ func TestProjectCommand(t *testing.T) {
 			commenter:       "sig-lead",
 			previousProject: "0.0.0",
 			previousColumn:  "Backlog",
-			expectedProject: "",
-			expectedColumn:  "",
+			expectedProject: "0.0.0",
+			expectedColumn:  "Backlog",
```

Comment:
Please add a test case for the new functionality ("Don't try to delete if the project doesn't exist").

Reply:
@cjwagner Thanks for the review and this is a good catch. I added test case to test the condition. I am a  bit unclear if we should use the issue number or the issue url when we print out the error. May be we do not need any of them, please let me know.